java/vector/src/test/java/org/apache/arrow/vector/TestBitVector.java

```diff
@@ -526,15 +505,17 @@ private void validateRange(int length, int start, int count) {
     try (BitVector bitVector = new BitVector("bits", allocator)) {
       bitVector.reset();
       bitVector.allocateNew(length);
-      bitVector.getMutator().setRangeToOne(start, count);
+      for (int i = start; i < start + count; i++) {
+        bitVector.set(i, 1);
+      }
```

Comment:
again, slightly different API, need to check range manually

Reply:
Should we consider add `setRangeToOne` back in `BitVector`?